src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -54,6 +54,23 @@ namespace winrt::TerminalApp::implementation
         TabViewItem().Header(_headerControl);
     }
 
+    // Method Description:
+    // - Called when the timer for the bell indicator in the tab header fires
+    // - Removes the bell indicator from the tab header
+    // Arguments:
+    // - sender, e: not used
+    void TerminalTab::_BellIndicatorTimerTick(Windows::Foundation::IInspectable const& /*sender*/, Windows::Foundation::IInspectable const& /*e*/)
+    {
+        auto weakThis{ get_weak() };
+
+        if (auto tab{ weakThis.get() })

```

Comment:
This will always pass. Since you constructed the event handler (line 290) with a weak reference, c++/winrt automatically ensures that you have a strong living reference inside this function. You can safely use `this` or just.. use unqualified members.